### ISSUE TITLE
[Bug] 3-dot menu does not fit inside its container

### ISSUE THREAD

**Issue by SlowNicoFish** [mozilla-mobile/fenix#19087]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Open any website
2. Open the 3 dot menu

### Expected behavior
The entire menu should fit in screen
### Actual behavior
The 3 dot menu is cut of and the user has to scroll down to do simple tasks such as go back or refresh
![Screenshot_20210417-013759__01](https://user-images.githubusercontent.com/19305293/115094829-3fe97880-9f1f-11eb-88a2-a5824a8254f1.jpg)

### Device information

* Device vendor / model and Android version: Oneplus 8T Oxygen OS 11
* Firefox for Android version: 210416 17:01 (Build #2015805003)

**Comment by cadeyrn** [comment]
It's by design that the menu is shorter by default (see #17796 for the implementation of the new menu). The row with the icons for back, forward, sharing and reload will be fixed once #19018 lands.

**Comment by mcarare** [comment]
Closing this as the behavior described fits UX specs and the remaining issues are tracked in other existing issues as per the [comment](https://github.com/mozilla-mobile/fenix/issues/19087#issuecomment-821808918) above.
